openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -106,6 +106,24 @@ <h3 class="vc-title vc-fade vc-polite-only">Pursue a verified certificate</h3>
                 % endif
             </main>
             <aside class="course-sidebar layout-col layout-col-a">
+                % if has_goal_permission:
+                    <div class="section section-goals ${'' if current_goal else 'hidden'}">
+                        <div class="current-goal-container">
+                            <span class="sr">${_("Set your course goal:")}</span>
```

Comment:
This is not needed here, the title should provide enough context for all users.

Reply:
Removed.